index2.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+
+    <title>David Underwood</title>
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <img class="cell-info" src="Light.png" alt="cell phone info" />
+      </div>
+      <div class="bottom-section">
+        <div class="welcome">
+          <a href="https://github.com/david63011">Welcome</a>
+        </div>
+        <img class="union" src="Union.png" alt="union photo" />
+      </div>
+    </header>
+    <div class="main-section">
+      <section>
+        <img src="background.png" alt="green background" />
+        <div class="greeting">
+          <p class="hi">Hey there. I'm</p>
+          <p class="first-name">Mariane</p>
+          <p class="job">I am a software</p>
+          <p class="job-info">Developer</p>
+        </div>
+        <div class="information-box">
+          <h1 class="info">I can help you build a product,</h1>
+          <h1 class="info2">feature or website Look through</h1>
+          <h1 class="info3">some of my work and experience!</h1>
+          <h1 class="info4">If you like what you see and have</h1>
+          <h1 class="info5">a project you need coded</h1>
+          <h1 class="info6">don’t hestiate to contact me.</h1>
+        </div>
+
+        <div class="icon-row">
+          <div class="tweet-connect">
+            <a href="https://twitter.com/david630"
+              ><img class="tweet" src="twitter.svg" alt="twitter picture"
+            /></a>
+          </div>
+
+          <img class="med" src="medium.svg" alt="medium picture" />
+          <div>
+            <a href="https://github.com/david63011"
+              ><img class="git" src="github.svg" alt="git picture"
+            /></a>
+          </div>
+
+          <img class="angel" src="angellist.svg" alt="angelist picture" />
+        </div>
+      </section>
+
+      <div class="top-one">
+        <h1 class="work">My Recent</h1>
+        <h1 class="work-2">Works</h1>
+      </div>
+
+      <img class="Indicator" src="Indicator.png" alt="green line" />
+
+      <div class="container">
+        <div class="column1"></div>
+        <div class="column2">
+          <div class="project-box">
+            <h1 class="project-name1">Multi-Post Stories</h1>
+            <h1 class="project-name-two">Gain+Glory</h1>
+          </div>
+          <div class="languages">
+            <div class="ruby">Ruby on Rails</div>
+            <div class="css">css</div>
+            <div class="js">JavScript</div>
+            <div class="html">HTML</div>
```

Comment:
- [ ] `required`
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag).
- Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.
- [ ] to the [html&Css](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) sematic tags

it should like something like this 
![image](https://user-images.githubusercontent.com/66230306/185713238-e06ed602-1aeb-4dff-87ea-f809a08d35e0.png)